mix.exs

```diff
@@ -24,16 +24,8 @@ defmodule Spacesuit.Mixfile do
   # Type "mix help compile.app" for more information
   def application do
     [
-      applications: [
-        :logger,
-        :cowboy,
-        :hackney,
-        :crypto,
-        :jose,
-        :exometer_newrelic_reporter,
-        :elixometer
-      ],
-      mod: {Spacesuit, []}
+      mod: {Spacesuit, []},
```

Comment:
As of 1.4(?) you no longer want to explicitly pass applications that need to be started. While this kind of sucks since it takes away the explicitness, it means that you can't mess up by forgetting to add a new application to the list when you start using it. In my case, I needed to clean this up or mox wasn't getting started.